src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs

```diff
@@ -641,6 +643,63 @@ public async Task<AuthenticationResult> GetAuthenticationResultForAppAsync(
             }
         }
 
+        private MergedOptions GetMergedOptions(string? authenticationScheme, TokenAcquisitionOptions? tokenAcquisitionOptions)
+        {
+            MergedOptions mergedOptions;
+
+            if (tokenAcquisitionOptions != null
+                && tokenAcquisitionOptions.ExtraParameters != null
+                && tokenAcquisitionOptions.ExtraParameters.TryGetValue("MicrosoftIdentityOptions", out object? identityOptions)
+                && identityOptions is MicrosoftEntraApplicationOptions microsoftEntraApplicationOptions)
+            {
+                MergedOptions parentMergedOptions = _tokenAcquisitionHost.GetOptions(authenticationScheme ?? tokenAcquisitionOptions?.AuthenticationOptionsName, out _);
+                mergedOptions = new MergedOptions()
+                {
+                    ClientId = microsoftEntraApplicationOptions.ClientId ?? parentMergedOptions.ClientId,
+                    Authority = microsoftEntraApplicationOptions.Authority != "//v2.0" ? microsoftEntraApplicationOptions.Authority : parentMergedOptions.Authority,
```

Comment:
is the `//` intended?

Reply:
yes. this is {instance}/{tenantId}/v2.0 with empty intance and tenantId